src/lib/y2storage/md.rb

```diff
@@ -287,6 +287,11 @@ def md_users
       to_storage_value.in_holders.to_a.map { |h| Storage.to_md_user(h) }
     end
 
+    # All Software RAIDs are Md devices, but MdMember derived class represents
+    # BIOS RAIDs. Class MdContainer is also derived from Md, but objects of this
+    # class are not be considered neither Software nor BIOS RAIDs.
+    # To properly exclude MdContainer as :raid device, here only Software RAIDs
+    # are considered to be raid type.
```

Comment:
NP: Wouldn't be too much to ask to have this explanation right on top of the `types << :raid if software_defined?` line instead of the heading of the method?